src/core/Pills/base.ts

```diff
@@ -0,0 +1,5 @@
+import { ButtonProps } from '@material-ui/core/Button';
+
+type PillsBaseProps = Omit<ButtonProps, 'variant'>;
```

Comment:
Pills don't need to extend from Button...try this instead
https://material-ui.com/components/chips/#chip
fix the `variant` from Mui to outlined and we'll have out own one...so Omit<ChipProps, 'variant'> will be the used